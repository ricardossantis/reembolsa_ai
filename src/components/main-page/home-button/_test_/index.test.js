import React from "react";
import renderer from "react-test-renderer";
import Button from "../index.jsx";

describe("Render Test", () => {
  it("renders a button", () => {
    const tree = renderer.create(<Button bckButton={"red"} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
