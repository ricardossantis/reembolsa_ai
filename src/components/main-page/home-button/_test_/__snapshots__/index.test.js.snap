// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders a button 1`] = `
<button
  className="sc-bdnylx iDLBqO"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "red",
    }
  }
>
  Quero participar
</button>
`;
